script/installer.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+const createWindowsInstaller = require('electron-winstaller').createWindowsInstaller
+const path = require('path')
+const rimraf = require('rimraf')
+
+deleteOutputFolder()
+  .then(getInstallerConfig)
+  .then(createWindowsInstaller)
+  .catch((error) => {
+    console.error(error.message || error)
+    process.exit(1)
+  })
+
+function getInstallerConfig() {
+  const rootPath = path.join(__dirname, '..')
+  const outPath = path.join(rootPath, 'release')
+
+  return Promise.resolve({
+    appDirectory: path.join(outPath, 'Mattermost-win32-ia32'),
```

Comment:
I somewhere read, that win64 is not supported. But I might mix up stuff, as I have read pretty much every readme of every installer project. 
Edit: Found it and remembered it the wrong way around. Only 64 bit it is. https://www.npmjs.com/package/electron-builder#auto-update